content/blog/osm-accepted-into-cncf-as-sandbox-project.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Open Service Mesh (OSM) accepted into CNCF as a sandbox project"
+slug: "open-service-mesh-osm-accepted-into-cncf-as-a-sandbox-project"
+authorname: "Phill Gibson"
+author: "@phillipgibson"
+authorlink: "https://twitter.com/phillipgibson"
+date: "2020-09-18T12:00:00-04:00"
+---
+
+We are delighted to announce that the Open Service Mesh (OSM) project has been accepted to enter the Cloud Native Computing Foundation (CNCF) as a [sandbox](https://www.cncf.io/sandbox-projects/) project.  
+
+Having Open Service Mesh be openly governed is a priority for the project maintainers, as was stated in the initial [launch blog](https://openservicemesh.io/blog/introducing-open-service-mesh/). The open governance structure provided by the CNCF allows developers across the ecosystem to easily contribute to the project under the auspices of the CNCF. 
+
+Now that OSM has met this milestone, we look forward to growing the community and continuing to provide a simple, stable, Envoy-based control plane that natively implements the complete set of [Service Mesh Interface (SMI) APIs](https://github.com/servicemeshinterface/smi-spec#service-mesh-interface-documents). Additionally, we hope organizations that have written their own in-house service mesh implementations consider collaborating on the project so that OSM can leverage the collective experience. 
+
+As for what comes next for OSM, the community is hard at work fully implementing the SMI APIs along with focusing on stability and end-to-end testing. You can follow the project [roadmap](https://github.com/openservicemesh/osm/blob/main/ROADMAP.md) for further detail. 
+
+If you’re interested in learning more about OSM, check out the initial [launch blog](https://www.cncf.io/sandbox-projects/). You can also check out the communication [channels](https://openservicemesh.io/) or the [Github repo](https://github.com/openservicemesh/osm) to familiarize yourself with the project. 

```

Comment:
The "launch blog" link here should go to https://openservicemesh.io/blog/introducing-open-service-mesh/ not https://www.cncf.io/sandbox-projects/